src/olympia/discovery/tests/test_admin.py

```diff
@@ -650,6 +667,16 @@ def test_can_add_with_discovery_edit_permission(self):
         assert item.custom_image == 'hero-featured-image/transparent.png'
         assert os.path.exists(os.path.join(
             settings.MEDIA_ROOT, 'hero-featured-image', 'transparent.png'))
+        assert os.path.exists(os.path.join(
+            settings.MEDIA_ROOT, 'hero-featured-image',
+            'thumbs', 'transparent.png'))
```

Comment:
lets use a larger image to test the resize works for the full size image too.  There are a few larger images in the same folder.